com.jetbrains.IntelliJ-IDEA-Ultimate.json

```diff
@@ -67,7 +68,7 @@
       "name": "ide-flatpak-wrapper",
       "buildsystem": "meson",
       "config-opts": [
-        "-Deditor_binary=/app/extra/IIU/bin/idea",
+        "-Deditor_binary=/app/extra/idea-IU/bin/idea",
```

Comment:
If you aren't able to use `FLATPAK_DEST` throughout the entire manifest, it's better to hard-code the path.

They say `FLATPAK_DEST` will never change, until the day it changes :sweat_smile:

Reply:
I can agree but what does it have to do with the indicated line?

Reply:
What I mean was that having a combination of `/app/...` and `FLATPAK_DEST`, really defeats the purpose of using an environment variable to refer to `/app`.

Just a suggestion.

I think the main reason `FLATPAK_DEST` cannot be used across the board in the Flatpak manifest is because it's not available/expanded/recognized in all cases.

Reply:
understood, FLATPAK_DEST is not perfect. just a way to change less line if folder change. and if in future it will be expanded i update it